### PR TITLE
Added count and details of in-scope in-use snapshots.

### PR DESCRIPTION
**Pull request by ei3kf**
Added count of in-scope in-use snapshots registered to an AMI to log file:
[2019-06-24 04:36:08,982][INFO]Total Snapshots in use with AMIs : 1146 
[2019-06-24 04:36:08,982][INFO]Total In-scope Snapshots currently in use with an AMI : 3

Added details of the in-scope in-use snapshots to log file:
[2019-06-24 04:35:38,898][INFO]Inscope Snapshot snap-0123456789 is currently in use with an AMI. ( Created by CreateImage(i-0123456789) for ami-0123456789 from vol-0123456789 )

### CODE REVIEW

File: SnapClean3.py [hunk 1]
```diff
@@ -266,6 +277,8 @@ def execute(self):
         # capture completion time
         finishTime = datetime.now().replace(microsecond=0)
 
+        self.logger.info('Total Snapshots in use with AMIs : %s', in_use_snapshots_count)
```
Comment (tangerinedream) [inline SnapClean3.py]: 1) I was confused by the name 'in_use_snapshot_count'.  Would you mind changing the name to 'snapshot_count_associated_with_AMIs' ?
2) like all of the other results, would you mind stashing  napshot_count_associated_with_AMIs in the results list, like we are doing for all of the other results we report on?  That will keep things consistent.
3) would you mind putting these two new totals after the existing 3 totals below?

**Review by tangerinedream — CHANGES_REQUESTED** [review]
Overall looks good.  Added an inline comment near line 281

File: SnapClean3.py (hunk 1 above)
Comment (ei3kf) [reply SnapClean3.py]: 1) Am no longer using *_count variables in the script.
results[TOTAL_SNAPSHOTS_ASSOCIATED_WITH_AMIS] and 
results[TOTAL_INSCOPE_SNAPSHOTS_ASSOCIATED_WITH_AMIS] are the count values.

2) Done.

3) Done.

[2019-06-25 04:01:14,854][INFO]Total Snapshots inspected 267
[2019-06-25 04:01:14,854][INFO]Expired Snapshots 133
[2019-06-25 04:01:14,854][INFO]Deleted Snapshots 0
**[2019-06-25 04:01:14,854][INFO]Total Snapshots in use with AMIs : 1144
[2019-06-25 04:01:14,854][INFO]Total In-scope Snapshots currently in use with an AMI : 3**
[2019-06-25 04:01:14,854][INFO]Exceptions Encountered 0

Let me know if this looks good or needs some more work?

**Review by tangerinedream — APPROVED** [review]
Excellent !

**Pull request merged**